tests/dummy/app/controllers/index.js

```diff
@@ -72,14 +104,40 @@ export default Controller.extend({
   networkFailure : false,
 
   actions : {
-    dragEndAction ({sourceList, sourceIndex, targetList, targetIndex}) {
+    dragEnd ({sourceList, sourceIndex, targetList, targetIndex}) {
       if (sourceList === targetList && sourceIndex === targetIndex) return
 
       const item = sourceList.objectAt(sourceIndex)
 
       sourceList.removeAt(sourceIndex)
       targetList.insertAt(targetIndex, item)
     },
+
+    determineForeignPosition ({draggedItem, items}) {
+      items = A(items.slice()) // create a copy of the list
```

Comment:
Wait, the latter is not true.

![image](https://user-images.githubusercontent.com/200884/28816100-3d93800c-76ac-11e7-8b40-9065681eca27.png)

This looks like Ember is violating its own conventions. :trollface: